Source/Eto.Gtk/Forms/GtkWindow.cs

```diff
@@ -249,7 +249,10 @@ public override void AttachEvent(string id)
 					Control.DeleteEvent += Connector.HandleDeleteEvent;
 					break;
 				case Eto.Forms.Control.ShownEvent:
-					Control.Shown += Connector.HandleShown;
+					Control.Shown += Connector.HandleShownEvent;
+					break;
+				case Window.OwnerChangedEvent:
```

Comment:
Changed Property-Event declaration to remove platform handling code in the next commit 7f696f6f9951299fa9c838c652e94b19b502b498.